packages/flutter/test/material/bottom_app_bar_theme_test.dart

```diff
@@ -10,6 +10,11 @@ import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
+  group('Material 2 tests', material2);
+  group('Material 3 tests', material3);
```

Comment:
Nit: I think usually we have just put the group functions inline rather than declaring things like material2 and material3 separately.  Not sure if this is a standardized thing or not, though.

Reply:
It is just a structural thing, it does allow quick navigation.
I also saw this somewhere in the flutter repo.

Reply:
I'll change this If it does need.

Reply:
Not a huge deal, but @justinmc is right that the common pattern is to just include the groups inline in the main function.